src/containerapp/azext_containerapp/_params.py

```diff
@@ -276,6 +276,11 @@ def load_arguments(self, _):
         c.argument('user_assigned', nargs='+', help="Space-separated user identities to be assigned.")
         c.argument('system_assigned', help="Boolean indicating whether to assign system-assigned identity.", action='store_true')
 
+    with self.argument_context('containerapp up', arg_group='Deploy a Foundry Model', is_preview=True) as c:
```

Comment:
Please add test to cover these new params

Reply:
Added in the latest commit. Run and passed locally with `azdev test test_containerapp_up_foundry_model_e2e --live`